src/engine/renderer/tr_local.h

```diff
@@ -246,7 +246,7 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 	  FRUSTUM_BOTTOM,
 	  FRUSTUM_TOP,
 	  FRUSTUM_NEAR,
-	  FRUSTUM_FAR,
+	  FRUSTUM_FAR = 5,
```

Comment:
I suspect this might be actually incorrect, but that's what the compiler really does.

Reply:
Maybe ad this as a comment?

Reply:
It seems to be intentional that FRUSTUM_FAR is often skipped in arrays or loops of length FRUSTUM_PLANES. Comparing our R_RecursiveWorldNode to [ioq3's](https://github.com/ioquake/ioq3/blob/bc7842e301f39df5cbe1bbb1da29831021fd7a7f/code/renderergl2/tr_world.c#L420), we can see that their (unrolled) version of the loop has 5 iterations, same as ours.

There is at least one minor mistake in the usage of FRUSTUM_PLANES though. The signature of R_CalcFrustumFarCorners suggests that an array of length 5 is a good input, but it reads from the 6th element.

Reply:
> There is at least one minor mistake in the usage of FRUSTUM_PLANES though. The signature of R_CalcFrustumFarCorners suggests that an array of length 5 is a good input, but it reads from the 6th element.

So, what should I do? I don't know what those FRUSTUM_PLANES planes are about, I only added that change to show explicitly what the code, in practice, does, and which might not be that obvious.

Reply:
You don't have to do anything.